addons/dashboard/2.0.x/dashboard-2.yaml

```diff
@@ -31,6 +31,8 @@ spec:
     chart: kubernetes-dashboard
     repo: https://mesosphere.github.io/charts/staging
     version: 2.0.0
+    valuesRemap:
+      "ingress.annotations.traefik\\.ingress\\.kubernetes\\.io/auth-url": "ingress.auth.auth-url"
```

Comment:
Ewww... that's ugly. Can't we do this with out `\\`

Reply:
I agree, but we can't unless we change the way valuesRemap works. The helm code (what kubeaddons is using to walk maps) uses "."s to traverse nested maps. It supports escaping `.`s with a `\`. YAML also supports escaping and requires a literal `\\` to be used for key names.

Reply:
Hmm.. actually removing the `""` around the key name allows for only one escape. At least with yq.